server_admin/topics/events/admin.adoc

```diff
@@ -1,36 +1,37 @@
 
 === Admin Events
 
-Any action an admin performs within the admin console can be recorded for auditing purposes.
-The Admin Console performs administrative functions by invoking on the {project_name} REST interface.  {project_name}
-audits these REST invocations.  The resulting events can then be viewed in the Admin Console.
+You can record all actions that are performed by an administrator in the Admin Console. The Admin Console performs administrative actions by invoking the {project_name} REST interface and {project_name} audits these REST invocations. You can view the resulting events in the Admin Console.
 
-To enable auditing of Admin actions, go to the `Events` left menu item and select the `Config` tab.
+To enable auditing of Admin actions:
 
+.Procedure
+. Click *Events* in the menu.
+. Click the *Config* tab.
++
 .Event Configuration
-image:{project_images}/login-events-config.png[]
-
-In the `Admin Events Settings` section, turn on the `Save Events` switch.
-
+image:{project_images}/login-events-config.png[Event Configuration]
++
+. Toggle *Save Events* to *ON* in the `Admin Events Settings` section. {project_name} displays the *Include Representation* switch.
```

Comment:
Sorry one more detail  -   `Admin Events Settings`  should use asterisks

Reply:
No worries, glad you spotted it.

Reply:
Now fixed.